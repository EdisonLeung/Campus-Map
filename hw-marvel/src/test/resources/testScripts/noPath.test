# Tests pathfinding on a node that has no edges and thus no path to another node

# Create Graph
CreateGraph noPath

# Add nodes
AddNode noPath a
AddNode noPath b
AddNode noPath c

# Add edges
AddEdge noPath b a BA
AddEdge noPath b c BC

# pathfinding from a to c
FindPath noPath a c