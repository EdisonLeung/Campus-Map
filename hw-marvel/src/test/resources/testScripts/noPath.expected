# Tests pathfinding on a node that has no edges and thus no path to another node

# Create Graph
created graph noPath

# Add nodes
added node a to noPath
added node b to noPath
added node c to noPath

# Add edges
added edge BA from b to a in noPath
added edge BC from b to c in noPath

# pathfinding from a to c
path from a to c:
no path found